rpcs3/Emu/Cell/Modules/cellGem.h

```diff
@@ -139,16 +139,16 @@ struct CellGemExtPortData
 
 struct CellGemImageState
 {
-	be_t<u64> frame_timestamp;
-	be_t<u64> timestamp;
-	be_t<f32> u;         // horizontal screen position in pixels
-	be_t<f32> v;         // vertical screen position in pixels
-	be_t<f32> r;         // size of sphere on screen in pixels
+	be_t<u64> frame_timestamp; // time the frame was captured by libCamera (usecs)
+	be_t<u64> timestamp;       // time processing of the frame was finished (usecs)
+	be_t<f32> u;               // horizontal screen position in pixels
+	be_t<f32> v;               // vertical screen position in pixels
+	be_t<f32> r;               // size of sphere on screen in pixels
 	be_t<f32> projectionx;
 	be_t<f32> projectiony;
-	be_t<f32> distance;
-	u8 visible;
-	u8 r_valid;
+	be_t<f32> distance;        // Move sphere distance from camera (probably)
+	u8 visible;	               // whether the sphere is visible in the current frame
```

Comment:
What's up with the whitespace here?

Reply:
no idea, file is good for me for have all line aligned

Reply:
NEVER use tabs for stuff like this, this is the result.

Reply:
ah ok

Reply:
@elad335 after check, velocity have used space here, so where is problem ?

Reply:
![image](https://user-images.githubusercontent.com/5261759/70389369-eeda8380-19be-11ea-8cc9-b95296240c4f.png)

Reply:
![image](https://user-images.githubusercontent.com/5261759/70389388-13366000-19bf-11ea-9dcb-39db0614beab.png)

Reply:
odd, it's misaligned for me.

Reply:
juust ignore it on gihub, is perfect

Reply:
![image](https://user-images.githubusercontent.com/23019877/70558434-301c9000-1b85-11ea-9656-6934ee62ca54.png)

Reply:
Ouch not see it lol to small